closure/goog/ui/tree/basenode.js

```diff
@@ -186,16 +186,16 @@ goog.ui.tree.BaseNode.prototype.initAccessibility = function() {
     }
 
     var img = this.getIconElement();
-    if (img) {
+    if (img && img.setAttribute) {
```

Comment:
I had the case that getExpandIconElement(), getIcontElement(), ... returned no real element like an &nbsp;  (which has not attribute or style).